MobileCenterDistribute/MobileCenterDistribute/Resources/ru.lproj/MobileCenterDistribute.strings

```diff
@@ -1,15 +1,15 @@
 /*
  * Update dialog buttons
  */
-"MSDistributeUpdateNow" = "Обновить сейчас";
-"MSDistributeAskMeInADay" = "Спросить завтра";
-"MSDistributeViewReleaseNotes" = "Просмотреть примечания к обновлению";
+"MSDistributeUpdateNow" = "Обновить";
+"MSDistributeAskMeInADay" = "Спросить через день";
+"MSDistributeViewReleaseNotes" = "Просмотреть заметки о выпуске";
 "MSDistributeClose" = "Закрыть";
 
 /*
  * Update dialog titles/messages
  */
-"MSDistributeAppUpdateAvailable" = "Доступно обновление";
-"MSDistributeAppUpdateAvailableMandatoryUpdateMessage" = "Вышло обязательное обновление. Вам необходимо обновиться до %@ %@ (%@).";
-"MSDistributeAppUpdateAvailableOptionalUpdateMessage" = "%@ %@ (%@) доступно для загрузки и установки.";
-"MSDistributeInAppUpdatesAreDisabled" = "Обновления из приложения отключены.";
+"MSDistributeAppUpdateAvailable" = "Доступно обновление приложения";
+"MSDistributeAppUpdateAvailableMandatoryUpdateMessage" = "Разработчики этого приложения требуют выполнить обновление до %@ %@ (%@).";
```

Comment:
This sounds much worse than before

Reply:
This is copied form TouchDown. I know this doesn't look good but it needs to be addressed in a separate PR. @jwallra please update the file in `Localization/`